packages/handbook-v1/en/tutorials/TS for the New Programmer.md

```diff
@@ -101,55 +103,69 @@ TypeScript, though, considers division of number by an array to be a nonsensical
 These may be legitimate problems with the code, or TypeScript being overly conservative.
-Throughout this guide we'll demonstrate how to add various TypeScript syntax to make these errors go away.
+Throughout this guide we'll demonstrate how to add various TypeScript syntax to eliminate such errors.
 
 #### Runtime Behavior
 
 TypeScript is also a programming language that preserves the _runtime behavior_ of JavaScript.
 For example, dividing by zero in JavaScript produces `Infinity` instead of throwing a runtime exception.
-TypeScript **never** changes the runtime behavior of JavaScript code.
+As a principle, TypeScript **never** changes the runtime behavior of JavaScript code.
 
-This means code you move from JavaScript to TypeScript is **guaranteed** to do the same thing, even if TypeScript thinks that code has type errors.
+This means that if you move code from JavaScript to TypeScript, it is **guaranteed** to run the same way, even if TypeScript thinks that the code has type errors.
 
 Keeping the same runtime behavior as JavaScript is a foundational promise of TypeScript because it means you can easily transition between the two languages without worrying about subtle differences that might make your program stop working.
 
+<!--
+Missing subsection on the fact that TS extends JS to add syntax for type
+specification.  (Since the immediately preceding text was raving about
+how JS code can be used in TS.)
+-->
+
 #### Erased Types
 
-TypeScript's type system is _erased_, meaning that once your code is compiled, there is no persisted type information in the resulting JavaScript code.
+Roughly speaking, once TypeScript's compiler is done with checking your code, it _erases_ the types to produced the resulting "compiled" code.
+This means that once your code is compiled, the resulting plain JS code has no type information.
 
-TypeScript also never changes the behavior of your program based on the types it inferred.
-While you might see more or fewer type errors, the type system itself has no bearing on how your program works once it's running.
+This also means that TypeScript never changes the _behavior_ of your program based on the types it inferred.
+The bottom line is that while you might see type errors during compilation, the type system itself has no bearing on how your program works when it runs.
 
 Finally, TypeScript doesn't provide any additional runtime libraries.
 Your programs will use the same standard library (or external libraries) as JavaScript programs, so there's no additional TypeScript-specific framework to learn.
+<!--
+Should extend this paragraph to say that there's an exception of
+allowing you to use newer JS features and transpile the code to an older
+JS, and this might add small stubs of functionality when needed.  (Maybe
+with an example --- something like `?.` would be good in showing readers
+that this document is maintained.)
+-->
```

Comment:
Yeah, I think a section on 'transpiling for older runtimes' is a good call here, it's likely someone with zero experience in programming would have no concept of this at all